src/utils/account.js

```diff
@@ -0,0 +1,16 @@
+import { getCurrentSanbaseSubscription } from './plans'
+import { PRO } from '../components/Navbar/NavbarProfileDropdown'
+
+export const checkIsProUser = data => {
+  const sanbaseSubscription = getCurrentSanbaseSubscription(data)
+
+  return sanbaseSubscription && sanbaseSubscription.plan
+    ? sanbaseSubscription.plan.name === PRO
```

Comment:
@GarageInc why not && sanbase.plan.name === PRO

Reply:
'copypaste'
Will change